pkgs/cargo-cp-artifact/package.json

```diff
@@ -1,17 +1,21 @@
 {
   "name": "cargo-cp-artifact",
-  "version": "0.1.7",
+  "version": "0.2.0",
   "description": "Copies compiler artifacts emitted by rustc by parsing Cargo metadata",
-  "main": "src/index.js",
+  "main": "index.node",
   "files": [
     "bin",
-    "src"
+    "index.node"
   ],
   "bin": {
     "cargo-cp-artifact": "bin/cargo-cp-artifact.js"
   },
   "scripts": {
-    "test": "mocha test"
+    "build": "cargo run -p cargo-cp-artifact -- -ac cargo-cp-artifact-adapter index.node -- cargo build --message-format=json-render-diagnostics",
```

Comment:
This works in tree, but won't work when installed from npm. The next step is now to implement pre-builds.